### ISSUE TITLE
script paths are incorrect for global CLI

### ISSUE THREAD

**Issue by ckundo** [ckundo/auto-vo#7]
running the CLI returns:

```Error: Command failed: ./osascripts/stopVoiceover.js
/bin/sh: ./osascripts/stopVoiceover.js: No such file or directory
```

[related to #6]

**Comment by ckundo** [comment]
fixed in `v1.2.2`
